VoodooI2CHID/VoodooI2CPrecisionTouchpadHIDEventDriver.hpp

```diff
@@ -49,6 +49,7 @@ class EXPORT VoodooI2CPrecisionTouchpadHIDEventDriver : public VoodooI2CMultitou
     IOReturn setPowerState(unsigned long whichState, IOService* whatDevice) override;
 
  protected:
+    IOReturn parseElements(UInt32 forUsage=0) override;
```

Comment:
Please replace the magic constant `0` with a header declaration or a proper equivalent from IO Kit.